lib/provider/camunda/parts/implementation/Listener.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(element, bpmnFactory, options, translate) {
 
       var event = (listener.get('event')) ? listener.get('event') : '<empty>';
 
-      var label = (event || '*') + ' : ' + (LISTENER_TYPE_LABEL[listenerType] || '');
+      var label = (event || '*') + ' : ' + translate(LISTENER_TYPE_LABEL[listenerType] || '');
```

Comment:
Have you tested this? I think it might be difficult to get the actual template string if we use a variable here. Let's keep the previous, explicit behaviour, shall we?

Reply:
Sure, I did.

Reply:
changed it.